src/spikeinterface/preprocessing/tests/test_whiten.py

```diff
@@ -41,6 +41,14 @@ def test_whiten(create_cache_folder):
     assert rec4.get_dtype() == "int16"
     assert rec4._kwargs["M"] is None
 
+    # test regularization
+    with pytest.raises(AssertionError):
```

Comment:
I do not see in the code wich assert is raised for this.
We should have better error than assert in this no ?

Reply:
I've used what was already done, open to suggestions